app/controllers/games_controller.rb

```diff
@@ -0,0 +1,48 @@
+class GamesController < ApplicationController
+
+
+
+  def index
+    @games = Game.all
+  end
+
+  def show
+    @game = Game.find(params[:id])
+    # @review = Review.new
+    # @reviews = Review.where(restaurant_id: @game.id)
+  end
+
+  def new
+    @game = Game.new
+  end
+
+
+  def edit
+    @game = Game.find(params[:id])
+  end
+
+  def create
+  @game = Game.new(game_params)
+
+    if @game.save
+      flash[:notice] = 'You have added a new game!'
+      redirect_to @game
+    else
+      @game.errors.full_messages.each do |error|
+        flash[:notice] = error
+      end
+      render :new
+    end
+  end
+
+  def destroy
```

Comment:
This method doesn't do anything. Avoid adding half-completed features since it becomes confusing for other developers to determine what is working and what isn't.